cmd/uid_provider/inprov/test.yaml

```diff
@@ -0,0 +1,7 @@
+uid: ghi
+password: 123456
+permissions:
+  "a.b":
+    - "get:ctype"
```

Comment:
this should be "get:type"

Reply:
You're right. This file was just for test, I deleted it